Source/carthage/Environment.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Environment.swift
+//  Carthage
+//
+//  Created by J.D. Healy on 2/6/15.
+//  Copyright (c) 2015 Carthage. All rights reserved.
+//
+
+import CarthageKit
+import Foundation
+import LlamaKit
+
+internal func getEnvironmentVariable(variable: String) -> Result<String> {
+	let environment = NSProcessInfo.processInfo().environment
+
+	if let value = environment[variable] as String? {
+		return success(value)
+	} else {
+		return failure(CarthageError.MissingEnvironmentVariable(variable: variable).error)
+	}
+}
+
+/// Information about the possible parent terminal.
+internal struct Terminal {
```

Comment:
I agree that it would be nice if this didn't simply contain static properties, but I think it's acceptable for the time being.